Integrations/GitHub/GitHub.py

```diff
@@ -17,33 +17,28 @@
 REPO = demisto.params().get('repository')
 BASE_URL = 'https://api.github.com'
 FETCH_INTERVAL = demisto.params()['fetch_interval']
-
+SUFFIX = {'list': '/repos/{}/{}/issues'.format(OWNER, REPO),
```

Comment:
No real need in dictionary here, but if you already did that with a dict please format it accordingly